docusaurus.config.js

```diff
@@ -89,21 +100,77 @@ module.exports = {
           label: 'Networks',
           to: 'networks/iota-1.5-chrysalis',
           activeBaseRegex: '^/networks/.*',
+          type: 'dropdown',
+          items: [
+            {
+              label: 'Chrysalis',
```

Comment:
```suggestion
              label: 'IOTA',
```

Reply:
I like how you just took my comment from above and fixed it :trollface: 
The reason I didn't fix it, was because I wasn't sure if we can use IOTA in the label and the first page on the new page talks about Chrysalis 😅 (Related to Muxxers comment. We need to work on our introduction docs and pages in general). Same for the Shimmer change

Reply:
We have to start somewhere :trollface: I'd say let's just fix it as we go, we are already piling up enough issues, wdyt?